lib/tasks/remove_verified_at_for_non_verified.rake

```diff
@@ -0,0 +1,65 @@
+namespace :profiles do
+  desc 'Remove verified_at if a profile is gpo or fraud pending'
+
+  ##
+  # Usage:
+  #
+  # Print errant profiles
+  # bundle exec rake profiles:remove_errant_verified_at _from_profiles
```

Comment:
Looks like the task name changed. Same on line 11.
```suggestion
  # bundle exec rake profiles:remove_verified_at_from_non_verified_profiles
```

Reply:
find and replace failed me :(